eos-phone-home

```diff
@@ -273,6 +285,54 @@ class PhoneHome(object):
 
         return False
 
+    def _get_mac_hash(self):
+        try:
+            ifaces = []
+            wireless = []
```

Comment:
Since both arrays hold same type of values (and especially since you use a local `iface` variable later, naming them similar might be clearer (i.e. `wlan_ifaces`, `eth_ifaces`)